### PR TITLE
hot-update-fix

### PR DESCRIPTION
**Pull request by meetdhananifynd**

### CODE REVIEW

File: src/helper/serve.utils.ts [hunk 1]
```diff
@@ -356,6 +356,9 @@ export async function startReactServer({ domain, host, isHMREnabled, port }) {
         if (request.url.indexOf('.hot-update.json') !== -1) {
             return response.json({ c: ['themeBundle'], r: [], m: [] });
         }
+        if (request.url.indexOf('.hot-update.js') !== -1) {
```
Comment (Shivraj0) [inline src/helper/serve.utils.ts:360]: Add brief comment here to explain why it is required.

**Review by Shivraj0 — COMMENTED** [review]
Reviewed.

File: src/helper/serve.utils.ts (hunk 1 above)
Comment (meetdhananifynd) [reply src/helper/serve.utils.ts:360]: added

**Review by Shivraj0 — APPROVED** [review]

**Review by meetdhananifynd — COMMENTED** [review]
/Filtering so that HMR file requests are not routed to skyfire pods

**Pull request merged**